test/download_response_test.mocks.dart

```diff
@@ -10,7 +10,7 @@ import 'package:mockito/mockito.dart' as _i1;
 // ignore_for_file: comment_references
 // ignore_for_file: unnecessary_parenthesis
 
-class _FakeUint8List extends _i1.Fake implements _i2.Uint8List {}
+// class _FakeUint8List extends _i1.Fake implements _i2.Uint8List {}
```

Comment:
This file should be generated by the `mockito` package (running `flutter pub run build_runner build` from the root of the project).
I believe that you manually edited this file in order to pass tests? 😕

Reply:
ups yeah, I forgot to uncomment that, sorry